src/frontend/apps/impress/src/features/docs/doc-share/component/DocShareModal.tsx

```diff
@@ -147,102 +162,139 @@ export const DocShareModal = ({ doc, onClose }: Props) => {
     });
   };
 
+  const getListHeight = () => {
+    const contentHeight = 690;
+    const inputHeight = 70;
+    const marginTop = 11;
+
+    const footerHeight = footerRef.current?.clientHeight ?? 0;
+    const selectedUsersHeight = selectedUsersRef.current?.clientHeight ?? 0;
+
+    return (
+      contentHeight -
+      footerHeight -
+      selectedUsersHeight -
+      inputHeight -
+      marginTop
+    );
+  };
+
```

Comment:
Not sure why you need a function, you can add this part inside handleRef, under the node condition, to avoid unecessary computation, wdyt ?